threeML/io/plotting/post_process_data_plots.py

```diff
@@ -319,39 +323,45 @@ def display_spectrum_model_counts(analysis, data=(), **kwargs):
 
         # Divide the various cases
 
-        if data._observation_noise_model == 'poisson':
+        if ratio_residuals:
+            residuals = (rebinned_observed_counts - rebinned_model_counts) / rebinned_model_counts
+            residual_errors = rebinned_observed_count_errors / rebinned_model_counts
+        else:
+            residual_errors = None 
+            if data._observation_noise_model == 'poisson':
 
-            if data._background_noise_model == 'poisson':
+                if data._background_noise_model == 'poisson':
 
-                # We use the Li-Ma formula to get the significance (sigma)
+                    # We use the Li-Ma formula to get the significance (sigma)
 
-                residuals = significance_calc.li_and_ma()
+                    residuals = significance_calc.li_and_ma()
 
-            elif data._background_noise_model == 'ideal':
+                elif data._background_noise_model == 'ideal':
 
-                residuals = significance_calc.known_background()
+                    residuals = significance_calc.known_background()
 
-            elif data._background_noise_model == 'gaussian':
+                elif data._background_noise_model == 'gaussian':
 
-                residuals = significance_calc.li_and_ma_equivalent_for_gaussian_background(rebinned_background_errors)
+                    residuals = significance_calc.li_and_ma_equivalent_for_gaussian_background(rebinned_background_errors)
 
-            else:
+                else:
 
-                raise RuntimeError("This is a bug")
+                    raise RuntimeError("This is a bug")
 
-        else:
+            else:
 
-            if data._background_noise_model is None:
+                if data._background_noise_model is None:
 
-                residuals = (rebinned_observed_counts - rebinned_model_counts) / rebinned_observed_count_errors
+                    residuals = (rebinned_observed_counts - rebinned_model_counts) / rebinned_observed_count_errors
 
-            else:
+                else:
 
-                raise NotImplementedError("Not yet implemented")
+                    raise NotImplementedError("Not yet implemented")
 
         residual_plot.add_data(mean_energy,
                                new_rate / new_chan_width,
                                residuals,
+                               residual_yerr=residual_errors,
                                yerr=new_err / new_chan_width,
                                xerr=delta_energy,
                                label=data._name,
```

Comment:
good point! Warning added. Feel free to reword.

Ratios are better than sigmas to asses relative shape of the residual structure, but there is no need to explain it to the user.